sys/include/net/ieee802154/radio.h

```diff
@@ -428,7 +428,8 @@ struct ieee802154_radio_ops {
      * @brief Read a frame from the internal framebuffer
      *
      * This function reads the received frame from the internal framebuffer.
-     * It should try to copy the received frame into @p buf
+     * It should try to copy the received PSDU frame into @p buf. The FCS
+     * field is ignored.
```

Comment:
```suggestion
     * field will **not** be copied and its size **not** be taken into account for the return value.
```

Maybe ignored is a bit vague.

Reply:
much better! thanks :)

Reply:
done! amended directly